citations-impl/impl/src/sql/oracle/sakai_citation.sql

```diff
@@ -28,7 +28,9 @@ CREATE TABLE CITATION_CITATION
 CREATE TABLE CITATION_COLLECTION_ORDER
 (
 	COLLECTION_ID VARCHAR2 (36) NOT NULL,
-	CITATION_ID VARCHAR2 (36) NOT NULL,
+	CITATION_ID VARCHAR (36),
+	SECTION_TYPE ENUM('heading1','heading2', 'heading3', 'description'),
+	VALUE TEXT,
 	LOCATION NUMBER(10,0) NOT NULL
```

Comment:
Is it worth adding a unique constraint on collection_id and location columns, so you can't end up with 2 items in the same position in the list?